src/form/Form.tsx

```diff
@@ -1443,16 +1437,25 @@ function Form({
       };
       if (submitData) {
         const submitButton = activeStep.buttons.find(findSubmitButton);
+        const authButton = activeStep.buttons.find(findAuthButton);
```

Comment:
let target = activeStep.buttons.find(findSubmitButton);
if (!target) target = activeStep.buttons.find(findAuthButton);

if (target && ...) buttonOnClick(target);

Reply:
yeah actually findAuthButton logic can be put into findSubmitButton